packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -724,15 +724,19 @@ class DescopeWc extends BaseDescopeWc {
           return;
         }
 
+        this.#handleSdkResponse(sdkResp);
```

Comment:
It's better to leave the handle response after the error check. The SDK relies on the "onError" callback for fatal errors

Reply:
Why does it matter?

Reply:
There's no use sending an error polling response to the native SDK, it serves no purpose.